_notebooks/2021-04-28-real-data-minka-zg-comparison.ipynb

```diff
@@ -0,0 +1,1067 @@
+{
```

Comment:
<p>too hard to see all of the dots here</p><p> </p><p>I also don't understand how ZG4 hack (yellow) could ever be lower than ZG1 hack (red). Seems like a bug to me? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neurodata/notebooks/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>